.github/actions/bootstrap-macos/action.yaml

```diff
@@ -25,6 +25,12 @@ runs:
       rm -rf "$(brew --prefix)/var/homebrew/locks"
       brew bundle --quiet
 
+      echo "-- install cmake"
+      curl -LO https://github.com/Kitware/CMake/releases/download/v3.31.7/cmake-3.31.7-macos-universal.tar.gz
+      tar -xzf cmake-3.31.7-macos-universal.tar.gz
+      sudo mv cmake-3.31.7-macos-universal /opt/cmake-3.31.7
+      echo "/opt/cmake-3.31.7/bin" >> "${GITHUB_PATH}"
```

Comment:
Wondering why this was needed, supposedly `cmake` is installed via brew (https://github.com/mobilecoinofficial/full-service/blob/main/Brewfile#L2)

Reply:
And supposedly there is a new cmake on homebrew: https://formulae.brew.sh/formula/cmake

Reply:
mbedtls-sys-auto v2.26.1 doesn't built with the latest CMake (or any version >= 4.0. CMake removed compatibility with versions of CMake older than 3.5 with the 4.0 release: https://cmake.org/cmake/help/latest/release/4.0.html#deprecated-and-removed-features